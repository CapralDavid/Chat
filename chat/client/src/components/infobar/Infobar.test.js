// import * as React from "react";
// import * as ReactDom from "react-dom";

// //когда есть эта строчка,то выдает ошибку
// //SyntaxError: Support for the experimental syntax 'jsx' isn't 
// //currently enabled (85:9):
//  import { Infobar } from "./Infobar.js";


import React from 'react';
import renderer from 'react-test-renderer';
import Infobar from './Infobar.js';

const three = 3;

test('render correct', () => {
    const component = renderer.create(
        <Infobar room="RoomOne"></Infobar>
      );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    

}); 



// const component = renderer.create(
//     <Link page="http://www.facebook.com">Facebook</Link>,
//   );