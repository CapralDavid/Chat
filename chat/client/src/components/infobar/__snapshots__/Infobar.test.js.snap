// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correct 1`] = `
<div
  className="infobar"
>
  <div
    className="leftInnContainer"
  >
    <img
      alt="ImageOnline"
      className="onlineIcon"
      src="test-file-stub"
    />
    <h3
      color="black"
    >
      RoomOne
    </h3>
  </div>
  <div
    className="rightInnContainer"
  >
    <a
      href="/"
    >
      <img
        alt="ImageClose"
        src="test-file-stub"
      />
    </a>
  </div>
</div>
`;
